client/src/containers/SongsProvider.js

```diff
@@ -1,13 +1,10 @@
 import {connect} from "react-redux";
-import {Songs} from "../components/Songs";
+import Songs from "../components/Songs";
 
-function mapStateToProps(state) {
-    return {
-        songs: state.songs
-    }
-}
+const mapStateToProps = state => ({
```

Comment:
Usually i would like
```js
const mapStateToProps = ({ songs }) => ({ songs });
```
But in this method you want to make it clear that what your'e getting is `state` so what you do here is fine